config/rbac/role.yaml

```diff
@@ -1,187 +1,13 @@
+---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  labels:
-    app.kubernetes.io/name: vm-operator
-    app.kubernetes.io/managed-by: kustomize
-  name: operator
+  name: manager-role
 rules:
-- apiGroups:
```

Comment:
Any reason to delete access to other components?

Reply:
it was removed automatically by kubebuilder, looks like it doesn't like config folder structure